QBWEBSDK-124

dimaspirit opened the pull request.

Commit:
Fixed bug in comments

Commit:
QBWEBSDK-124
 - without time tracking

Commit:
- timer

Commit:
merged

soulfly commented:
1) acceptCallTime не нужно делать глобальным, ты его используешь только в WebRTCSession.prototype.accept
2) startCallTime лучше инициализировать не в _createPeer, а в _callInternal и processOnCall. Иначе эта переменная у тебя инициализируется N paз (N=количество оппонентов)
3) Переименуй _startOfferTimer и offerTimer в _startWaitingOfferOrAnswerTimer и waitingOfferOrAnswerTimer
4) У тебя нет кода, который кенселит этот таймер если другой юзер все-таки сделал accept

soulfly commented:
5) Тут еще можно закрывать пир

```
        if(peerConnection.state !== RTCPeerConnection.State.CONNECTED) {
              peerConnection._clearDialingTimer();
              peerConnection.release();
              self.processOnNotAnswer(peerConnection);
        }        
```

soulfly commented:
6) Вообще у нас много где есть места с кодом

```
   peerConnection._clearDialingTimer();
   peerConnection.release();
```

нужно сделать спец функцию типа closePeerConnection(peer)

Commit:
fixed review

Commit:
Merge branch 'develop.webrtc.QBWEBSDK-124_v2' of https://github.com/QuickBlox/quickblox-javascript-sdk into develop.webrtc.QBWEBSDK-124_v2

Conflicts:
	js/modules/webrtc/qbWebRTCSession.js
	quickblox.min.js

Commit:
l f

dimaspirit commented:
1. Ответ в комментарии
2. _createPeer потому что нужно дата создания последнего пира.
   _callInternal и processOnCall не подходят для инициализации.
   processOnCall - циклично вызывается,
   _callInternal - инициализуется у инициатора звонка;
3. fixed 
4. fixed
5. Метод processOnNotAnswer закрывает сессию и очищает таймер.
   http://joxi.ru/Drlaw0pUpVbWmP

Commit:
Merge remote-tracking branch 'remotes/origin/develop.webrtc' into develop.webrtc.QBWEBSDK-124_v2

Conflicts:
	quickblox.min.js

Commit:
cleanup

Commit:
merged

Commit:
merged

Commit:
add stopTimer onprocessOnReject

Commit:
Merge branch 'develop.webrtc.QBWEBSDK-124_v2' of https://github.com/QuickBlox/quickblox-javascript-sdk into develop.webrtc.QBWEBSDK-124_v2

Conflicts:
	quickblox.min.js

Commit:
delete _isUserLast

Commit:
cleanup

Commit:
Merge branch 'develop.webrtc.QBWEBSDK-124_v2' of github.com:QuickBlox/quickblox-javascript-sdk into develop.webrtc.QBWEBSDK-124_v2

Commit:
removed quickblox.js

Commit:
cleanup

Commit:
move call of _startWaitingOfferOrAnswerTimer

Commit:
fixes for answerTimeInterval

Commit:
finished
